baselines/diabetic_retinopathy_detection/jax_finetune_deterministic.py

```diff
@@ -515,7 +519,7 @@ def decay_fn(v, wd):
         results_arrs['y_true'] = np.concatenate(results_arrs['y_true'], axis=0)
         results_arrs['y_pred'] = np.concatenate(
             results_arrs['y_pred'], axis=0).astype('float64')
-        results_arrs['y_pred_entropy'] = vit_utils.entropy(
+        results_arrs['y_pred_entropy'] = entropy(
```

Comment:
Let's keep with the call to vit_utils. Internally, I had to do this because SciPy uses an older version internally that doesn't support the `axis` argument, so I copied the function.